meta/3rd/lfs/library/lfs.lua

```diff
@@ -0,0 +1,162 @@
+    lfs.setmode (file, mode)
+    Sets the writing mode for a file. The mode string can be either "binary" or "text". Returns true followed the previous mode string for the file, or nil followed by an error string in case of errors. On non-Windows platforms, where the two modes are identical, setting the mode has no effect, and the mode is always returned as binary.
+]]
+---@param dirname string
+---@return boolean, string
+function lfs.mkdir(dirname)
+end
+
+--[[
+    Removes an existing directory. The argument is the name of the directory.
+    Returns true in case of success or nil, an error message and a system-dependent error code in case of error.
+]]
+---@param dirname string
+---@return boolean, string
+function lfs.rmdir(dirname)
+end
+
+--[[
+Sets the writing mode for a file. The mode string can be either "binary" or "text". Returns true followed the previous mode string for the file, or nil followed by an error string in case of errors. On non-Windows platforms, where the two modes are identical, setting the mode has no effect, and the mode is always returned as binary.
+]]
+---@param file string
+---@param mode 'binary'|'text'
+---@return boolean, string
+function lfs.setmode(file, mode)
+end
+
+--[[
+Identical to lfs.attributes except that it obtains information about the link itself (not the file it refers to). It also adds a target field, containing the file name that the symlink points to. On Windows this function does not yet support links, and is identical to lfs.attributes.
+]]
+---@param filepath string
+---@param request_name? LuaFileSystem.AttributeName
+---@return LuaFileSystem.Attributes
+function lfs.symlinkattributes(filepath, request_name)
+end
+
+--[[
+Set access and modification times of a file. This function is a bind to utime function. The first argument is the filename, the second argument (atime) is the access time, and the third argument (mtime) is the modification time. Both times are provided in seconds (which should be generated with Lua standard function os.time). If the modification time is omitted, the access time provided is used; if both times are omitted, the current time is used.
+Returns true in case of success or nil, an error message and a system-dependent error code in case of error.
+]]
+---@param filepath string
+---@param atime? integer
+---@param mtime? integer
+---@return boolean, string
+function lfs.touch(filepath, atime, mtime)
+end
+
+--[[
+Unlocks a file or a part of it. This function works on open files; the file handle should be specified as the first argument. The optional arguments start and length can be used to specify a starting point and its length; both should be numbers.
+Returns true if the operation was successful; in case of error, it returns nil plus an error string.
+]]
+---@param filehandle file*
+---@param start? integer
+---@param length? integer
+---@return boolean, string
+function lfs.unlock(filehandle, start, length)
+end
```

Comment:
Can you please add a `return` to the end of this file?

```suggestion
end

return lfs
```

This way `local lfs = require "lfs"` will work properly.